Samples/TestStack.BDDfy.Samples/ModuleInitializer.cs

```diff
@@ -0,0 +1,21 @@
+﻿using TestStack.BDDfy.Configuration;
+using TestStack.BDDfy.Reporters.Html;
+using TestStack.BDDfy.Samples;
+using TestStack.BDDfy.Samples.Atm;
+
+/// <summary>
+/// Used by the ModuleInit. All code inside the Initialize method is ran as soon as the assembly is loaded.
+/// </summary>
+public static class ModuleInitializer
```

Comment:
I really wish xUnit would add this really critical missing bit. It is VERY annoying.